mdx/guides/understanding-rfqt.mdx

```diff
@@ -0,0 +1,62 @@
+In its role as a DEX aggregator, the 0x API integrates both on- and off-chain liquidity. On-chain liquidity is sourced by sampling smart contract liquidity pools, such as Uniswap and Curve. Off-chain liquidity has historically been sourced only by surveying open orderbooks on 0x Mesh.
+
+With the use of RFQ-T, the API aggregates not just on-chain pools and Mesh order books, but also real-time quotes from live market makers offering exclusive pricing.
+
+RFQ is an abbreviation of Request for Quote, in that the 0x API requests quotes from market makers on behalf of its clients. The -T suffix indicates that the settlement transaction is to be submitted to the blockchain by the Taker. (An RFQ-M model, in which the maker submits the transaction, is under consideration, but is outside the scope of this Guide.)
+
+## Limitations of Open Liquidity
+
+Market makers providing open liquidity are subject to unique risks that impact their bottom-line pricing. If asset prices exhibit great movement in the markets, outstanding orders can flip from profitable to loss-incurring. And cancelling outstanding orders can be expensive and unreliable, subject to gas markets and front-running. Carrying these risks has a real cost, and that cost effectively deteriorates pricing for takers.
+
+In practice, such losses are largely at the hands of one particular class of takers: arb bots. If makers share orders with an unspecified/null taker address, as is typical in Mesh order sharing, then those orders can be filled by anyone, including arb bots. Human traders, on the other hand, are extremely unlikely to be able to take advantage of such arbitrage opportunities, and if market makers could somehow serve orders only to those human users, they could afford to offer better prices to them.
+
+## Trusted Takers
+
+The 0x API now automatically attempts to supplement every quote with RFQ-T liquidity. The biggest determinant of whether such live quotes will be aggregated with other liquidity sources is whether the client is a known and trusted taker
+
+Trusted takers are identified by an API key, specified in the HTTP header `0x-api-key`. In addition, to qualify for RFQ-T liquidity, the request must also specify a non-null `takerAddress` query parameter, which will be included in the order that the maker signs, preventing anyone else from filling that order.
```

Comment:
This is kind of repetitive and already covered in `mdx/api/index.mdx`, have you considered having 1 document link to the other? this way, we don't have to maintain content on both pages.